torch/csrc/jit/python/init.cpp

```diff
@@ -1349,6 +1349,12 @@ void initJITBindings(PyObject* module) {
                 auto snb = toSymIntNode(a, b);
                 return a->sub(snb);
               })
+          .def(
+              "__rsub__",
+              [](c10::SymIntNode a, py::object b) -> c10::SymIntNode {
+                auto snb = toSymIntNode(a, b);
+                return a->sub(snb);
```

Comment:
You sure about this? It looks reversed of what it should be

Reply:
no, typo, I believe.